gradle/test.gradle

```diff
@@ -86,6 +92,7 @@ task runTestsWithTag(type: Test) {
         // Running them in parallel will cause runners to clobber this state, so these tests must be run serially.
         maxParallelForks = (testTag == 'unit' ? localCores : 1)
```

Comment:
This condition also needs an update, something like `! testTag.startsWith('integration')` should work

Reply:
Done